examples/pytorch/AxHyperOptimizationPTL/iris_data_module.py

```diff
@@ -15,6 +15,9 @@ def __init__(self):
         self.columns = None
 
     def prepare_data(self):
+        pass
```

Comment:
Maybe we could remove this if `prepare_data` is not an abstract method that must be implemented (not a big issue though).

Reply:
The official doc for `prepare_data`:

https://pytorch-lightning.readthedocs.io/en/latest/extensions/datamodules.html#prepare-data

Reply:
@harupy Let's see if we can safely remove it. Looks like `LightningDataModule` inherits from `DataHooks`, which defines `prepare_data` but does not designate it abstract.

Reply:
@dbczumar Thanks for the review! Got it. I'll remove them.